topics/signer_standards_overview.md

```diff
@@ -0,0 +1,187 @@
+
+    RP ->> S: Request permission / information
+    S ->> U: Display connection details (requested networks, scopes)<br/> Ask to select information to share with Relying Party
+    U ->> S: Select information<br/>Approve request
+    S ->> S: Store the granted permission scopes
+    S ->> RP: Permission / information response
+```
+This diagram has been simplified, see [ICRC-25](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/icrc_25_signer_interaction_standard.md) for the full flow.
+
+### Use-Case 2: Relying Party Initiated Transaction Approval Flow
+
+In this use-case the relying party initiates canister calls (transactions) on behalf of a signer controlled identity.
+The user can approve or reject the transaction on the signer UI. After executing the canister call, the signer must
+return the result to the relying party.
+
+This flow is the main use-case for [ICRC-25](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/icrc_25_signer_interaction_standard.md). It offers an alternative interaction model to the currently used session delegations.
+In particular, it allows multiple relying parties to securely interact with the IC using the same signer controlled identity
+without being restricted to a disjoint set of target canisters.
+
+In the future, batch transactions could be a possible extensions to this flow.
+
+The following diagram shows the flow:
+
+```mermaid
+sequenceDiagram
+    participant RP as Relying Party
+    participant S as Signer
+    participant U as User
+    participant C as Target Canister
+
+    RP ->> S: Request canister call
+    S ->> C: Request consent message
+    C ->> S: Consent message response
+    S ->> U: Display consent message<br>(Transaction details)
+    U ->> S: Approve request
+    S ->> C: Submit canister call
+    S ->> S: Poll for canister call result
+    S ->> U: Display success message
+    S ->> RP: Canister call response
+```
+This diagram has been simplified, see [ICRC-25](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/icrc_25_signer_interaction_standard.md) and [ICRC-21](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/icrc_21_consent_msg.md) for more details.
+
+## Signer Architecture
+
+The signer architecture is not part of the standards. Any component that fulfills the following requirements can act as a signer:
+- Ability to sign messages with an IC supported signature scheme
+- Ability to verify messages signed with an IC supported signature scheme
+- Ability to interact with the IC to send messages and receive responses
+- Implementation of an appropriate transport channel to communicate with a relying party
+
+User interaction is not mandated as developers should be able to build non-interactive signers. However, most common
+implementation  is expected to be a signer that is interactive and displays transaction information to the user and 
+receives user input synchronously.
+
```

Comment:
maybe add a sub-section like 
"### Example types of compliant signers" that contains all examples below?